.github/workflows/custom_docker_builds.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           - docker-image: ./images/build-timing-processor
             image-tags: ghcr.io/spack/build-timing-processor:0.0.4
           - docker-image: ./analytics
-            image-tags: ghcr.io/spack/django:0.0.1
+            image-tags: ghcr.io/spack/django:0.0.2
```

Comment:
Don't the references need to be updated for this as well?

Reply:
The celery runners also override this command and probably need to run migrate as well. What do you think?

Reply:
Good catch, I totally forgot about the other references. Regarding the command overwriting in celery as well, would it make sense to move the call to `migrate` into the entrypoint? Essentially using the entrypoint script pattern shown [here](https://stackoverflow.com/a/41518225).

Reply:
(testing that pattern locally seems to work as intended)

Reply:
Sounds good to me.